src/QuickMarcEditor/QuickMarcEditorRows/utils.js

```diff
@@ -17,3 +17,7 @@ export const hasIndicatorException = recordRow => INDICATOR_EXEPTION_ROWS.includ
 const ADD_EXCEPTION_ROWS = [LEADER_TAG, '001', '005', '008'];
 
 export const hasAddException = recordRow => ADD_EXCEPTION_ROWS.includes(recordRow.tag);
+
+const DELETE_EXCEPTION_ROWS = [LEADER_TAG, '001', '005', '008', '999'];
```

Comment:
ticket says about `999ff`, please look at isReadOnly - seems you'll need to add common function